game_eggs/minecraft/java/glowstone/egg-glowstone.json

```diff
@@ -0,0 +1,89 @@
+            "container": "eclipse-temurin:8-jdk",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Server Jar File",
+            "description": "The name of the server jarfile to run the server with.",
+            "env_variable": "SERVER_JARFILE",
+            "default_value": "glowstone.jar",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|regex:\/^([\\w\\d._-]+)(\\.jar)$\/"
+        },
+        {
+            "name": "Glowstone Version",
+            "description": "The version of Glowstone to download. If it ends in SNAPSHOT, it will attempt to download from the maven repository.",
+            "env_variable": "VERSION",
+            "default_value": "latest",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:20"
+        },
+        {
+            "name": "GitHub User",
+            "description": "GitHub user to use for api calls.\r\n\r\nThis only needs to be set if you hit the GitHub API too often across multiple servers.",
+            "env_variable": "GITHUB_USER",
+            "default_value": "",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "string|nullable"
+        },
+        {
+            "name": "GitHub OAuth Token",
+            "description": "This can be either an OAuth or a Personal Access Token.\r\n\r\nThis is required for the install is you set a user.",
+            "env_variable": "GITHUB_OAUTH_TOKEN",
+            "default_value": "",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "string|nullable"
+        },
+        {
+            "name": "GitHub Package",
```

Comment:
These Github release match variables aren't necessary for end-users. It can be set in the install script instead, which makes it cleaner and prevents someone from breaking the whole install

Reply:
I left this in in case someone (like myself) wants to use a fork on GitHub for development purposes or just running it.